api/host.go

```diff
@@ -190,9 +191,9 @@ type (
 	}
 
 	HostCheck struct {
-		Gouging   HostGougingBreakdown   `json:"gouging"`
-		Score     HostScoreBreakdown     `json:"score"`
-		Usability HostUsabilityBreakdown `json:"usability"`
+		GougingBreakdown   HostGougingBreakdown   `json:"gougingBreakdown"`
+		ScoreBreakdown     HostScoreBreakdown     `json:"scoreBreakdown"`
+		UsabilityBreakdown HostUsabilityBreakdown `json:"usabilityBreakdown"`
```

Comment:
Open to better names for those fields. But we can't have an overall `Score` while also calling the breakdown `Score`. So the idea was to make `Score` and `Usable` the "summary" of the breakdown fields.

Reply:
Sounds good